aten/src/ATen/DLConvertor.cpp

```diff
@@ -425,10 +425,20 @@ at::Tensor fromDLPackImpl(T* src, std::function<void(void*)> deleter) {
   Device device = getATenDevice(dl_tensor.device.device_type, dl_tensor.device.device_id, dl_tensor.data);
   ScalarType stype = toScalarType(dl_tensor.dtype);
 
+  int64_t storage_offset = 0;
+  if(dl_tensor.byte_offset > 0) {
+    size_t element_size = c10::elementSize(stype);
+    TORCH_CHECK(
```

Comment:
Should be TORCH_CHECK_VALUE or TORCH_CHECK_ASSERT don't you think. Implies input either has invalid input (first case) or violates a specific assertion invariant. Since the input is user provided, would lean to first one